my-app/src/App.js

```diff
@@ -28,12 +30,15 @@ function App() {
         <div className="main">
           <SideBar isActive={isActive}></SideBar>
           <Switch>
+            <Route path='/RegisterNewUser'
```

Comment:
Maybe it's better to create global object 
Like: 
const ROUTES  = {
 regirster: "/RegisterNewUser"
}
<Route path={ROUTES.register}/>

Reply:
Why RegisterNewUser? maybe it's better to /register in lower case

Reply:
/singup - the best way